tests/end-to-end/api/05-chat.js

```diff
@@ -32,7 +32,7 @@ describe('[Chat]', function() {
 					author_icon: 'https://avatars.githubusercontent.com/u/850391?v=3',
 					title: 'Attachment Example',
 					title_link: 'https://youtube.com',
-					title_link_download: 'https://rocket.chat/download',
+					title_link_download: true,
```

Comment:
I think we can have more tests here, with lots of different variations and some fail case. Can you do that, please?